frontend/app-preview/src/views/LandingPage.module.css

```diff
@@ -45,6 +45,10 @@
   font-family: var(--font-family);
 }
 
+.popOver {
+  max-width: 320px;
+}
```

Comment:
"Popover" er ett ord;
```suggestion
.popover {
  max-width: 320px;
}
```
Hvor kommer `320px` fra, egentlig? Er dette nok for å sørge for at knappene fyller bredden?

Reply:
Blir sånn her

<img width="484" alt="Screenshot 2023-10-31 at 07 35 12" src="https://github.com/Altinn/altinn-studio/assets/71079896/794fb292-e665-46c9-8cfa-89b35d7beaf1">

Reply:
Vi bør unngå å bruke absolutte størrelser på denne måten fordi det er avhengig av font og innholdet i teksten. Dette kan endre seg i koden, og vi må ta høyde for at brukeren kan endre innstillingene for skriftstørrelse i nettleseren. I stedet bør vi prøve å la bredden være styrt av knappene direkte - det kan vi få til ved hjelp av [grid](https://css-tricks.com/snippets/css/complete-guide-grid/). Her er et forslag i ren HTML som du kan bruke som utgangspunkt:

HTML:
```
<div class="grid">
    <p class="message">Ønsker du en påminnelse neste gang du logger på?</p>
    <button class="button yes">Ja, takk</button>
    <button class="button no">Nei, ikke vis denne meldingen igjen</button>
</div>
```
CSS:
```
.grid {
    display: grid;
    grid-template-columns: min-content min-content;
    gap: .5rem;
}
.message { grid-area: 1 / 1 / 2 / 3; margin: 0 }
.yes { grid-area: 2 / 1 / 3 / 2 }
.no { grid-area: 2 / 2 / 3 / 3 }
.button { white-space: nowrap }
```

Her har jeg brukt `grid-template-columns` til å definere to kolonner som begge skal være så smale som mulig. Deretter har jeg brukt `grid-area` til å definere hvilket område av tabellen de forskjellige elementene skal vises i. Ved å sette `white-space: nowrap` på knappene, blir teksten tvunget til å være på én linje, og dermed blir det bredden som kolonnen justerer seg etter.

I tillegg har jeg satt inn `gap` for å definere lik avstand mellom alle elementene.